### ISSUE TITLE
Write migration tutorial

### ISSUE THREAD

**Issue by Make42** [kieferk/dfply#32]
On my development machine I installed dfply version 0.2.4. Now I installed dfply with `pip install dfply` on the server and suddenly I got `NameError: name 'groupby' is not defined`. It took me ages to realise it's not that code that is wrong, but you had a new release (which was such a surprise because it had been a while). It seems that your `groupby` is now `group_by`... maybe you can write a migration tutorial? [Tensorflow did it reasonably well.](https://www.tensorflow.org/install/migration)

**Comment by kieferk** [comment]
Yes I changed the `grouby` to be `group_by` to mirror the `dplyr` function. Apologies for the code-breaking change. One way to resolve this in your code is to have an `from dfply.group import group_by as groupby`.

Given that is the only function name that changed between v2 and v3 (as far as I recall) a migration tutorial wouldn't have anything in it except that, but I will add something at the top of the readme to notify people.

**Comment by Make42** [comment]
Thank you very much!

Also thank you very much for continuing development! It it much appreciated.

**Comment by Make42** [comment]
The code examples in the README.md are still on `groupby` - once again I got confused when reading the examples... ;-)
